apps/web/pages/signup.tsx

```diff
@@ -276,25 +268,29 @@ export default function Signup({
                 await signUp(updatedValues);
               }}>
               {/* Username */}
-              <UsernameField
-                label={t("username")}
-                username={watch("username")}
-                premium={premiumUsername}
-                usernameTaken={usernameTaken}
-                setUsernameTaken={(value) => setUsernameTaken(value)}
-                data-testid="signup-usernamefield"
-                setPremium={(value) => setPremiumUsername(value)}
-                addOnLeading={
-                  orgSlug
-                    ? `${getOrgFullOrigin(orgSlug, { protocol: true })}/`
-                    : `${process.env.NEXT_PUBLIC_WEBSITE_URL}/`
-                }
-              />
+              {!isOrgInviteByLink ? (
+                <UsernameField
+                  label={t("username")}
+                  username={watch("username") || ""}
```

Comment:
`username` is optional at client now because it's requirement depends upon if it's an org case or non-org case. It is separately handled now at backend.